### ISSUE TITLE
dns/dyndns: phpDynDNS.inc warnings

### ISSUE THREAD

**Issue by larseggert** [opnsense/plugins#1857]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

[x] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md

[x] I have searched the existing issues and I'm convinced that mine is new.

[x] The title contains the plugin to which this issue belongs

**Describe the bug**
I see these lines in the log every night since configuring dynamic DNS with cloudflare (v4 and v6):
```
[27-May-2020 00:21:20 Europe/Helsinki] PHP Warning:  Invalid argument supplied for foreach() in /usr/local/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc on line 775
```

**To Reproduce**
Unsure. Set up v4 and v6 dynamic DNS with Cloudflare?

**Expected behavior**
Not to see these warnings.

**Comment by AdSchellevis** [comment]
duplicate https://github.com/opnsense/plugins/issues/1564 ?

**Comment by larseggert** [comment]
Possibly! I'm closing this for now, will reopen if it's not fixed.

**Comment by seitzbg** [comment]
Not fixed in OPNsense 20.1.7 bb116282f

**Comment by fichtner** [comment]
It appears #1564 is still open so no need to raise concern about it in closed tickets
